ci/steps.lib.yml

```diff
@@ -134,11 +130,7 @@ config:
     dir: #@ input_name
     args:
     - "-c"
-    #@ if is_debug:
-    - "make $MAKE_HELP debug all run-tests"
```

Comment:
We don't need this line or the `debug` target any more, as we now run with this code, called a few lines earlier:
```
116     #@ if is_debug:
117     BUILD_MODE: "debug"
118     #@ end
119
```